docs/sections/user_guide/cli/tools/mode_template.rst

```diff
@@ -237,8 +248,6 @@ and a YAML file called ``values.yaml`` with the following contents:
      $ uw template render --input-file template --values-file values.nml
      Hello, World!
 
-  Note that ``ini`` and ``nml`` configs are, by definition, depth-2 configs, while ``sh`` configs are depth-1, and ``yaml`` configs have arbitrary depth.
```

Comment:
This line seemed out of place to me -- there's no other discussion of depth here, and it wasn't clear to me what the reader was supposed to make of this statement.

Reply:
We do need to explain something about the depth limits in the documentation, but it is awkward here. I think we still need it, but it should move elsewhere and rephrased